osu.Game/Localisation/FirstRunOverlayImportFromStableScreenStrings.cs

```diff
@@ -51,11 +51,21 @@ public static class FirstRunOverlayImportFromStableScreenStrings
         /// </summary>
         public static LocalisableString Items(int arg0) => new TranslatableString(getKey(@"items"), @"{0} item(s)", arg0);
 
+        /// <summary>
+        /// "Data migration will use &quot;copy on write&quot;. No extra disk space will be used, and you can delete or modify either data folder at any point without affecting the other installation."
+        /// </summary>
+        public static LocalisableString DataMigrationNoExtraSpaceWithCoW => new TranslatableString(getKey(@"data_migration_no_extra_space_cow"), @"Data migration will use ""copy on write"". No extra disk space will be used, and you can delete or modify either data folder at any point without affecting the other installation.");
```

Comment:
Yes, I agree that it's less important to end users about the technical details.
What's I'm unsure is the risk about editing with hard links, for example someone edited the corresponding file from stable Songs folder. Informing the incorporation with CoW may satisfy the techies. Of course these can be put into the wiki.

Reply:
> What's I'm unsure is the risk about editing with hard links, for example someone edited the corresponding file from stable Songs folder

What risk are you referring to? There is no risk as long as edits are performed through lazer.

Reply:
> edits are performed through lazer

This is quite a strong statement. For example, user may continue to edit beatmaps in stable after importing.

For me, hard linking is immediately questionable with non-protected files.

Reply:
> For example, user may continue to edit beatmaps in stable after importing.

Should be fine too. Last I checked editing via stable also caused no issues. It was only manual editing that posed a potential issue.

Reply:
Manual editing is a thing people often do.

I'm agreed to use one single message and updated the code. However I still think the term "hard link" should be revised, since it's more a specialized terminology, which can cause concern for really advanced users.